### PR TITLE
fix: start your journey button fixed

### PR DESCRIPTION
**Pull request by sumanjeet0012**
## Related Issue

fix: #134 
## Description

Now the 'start your journey' button changes to 'Checkout your feed' button and redirects logged in unverified users to feed page.

## Screenshots

image

### CODE REVIEW

File: src/components/pages/home/index.tsx [hunk 1]
```diff
@@ -170,7 +170,7 @@ function HomePage() {
             </p>
 
             <div className="flex justify-center md:justify-start">
-              {state?.creds.userId && state?.creds.isVerified ? (
+              {state?.creds.userId && state?.creds.userId !== '' ? (
```
Comment (Sanchitbajaj02) [inline src/components/pages/home/index.tsx:173]: isn't isVerified not coming there?

**Review by Sanchitbajaj02 — CHANGES_REQUESTED** [review]

**Comment by sumanjeet0012** [comment]
what changes should I make in this issue

**Comment by Sanchitbajaj02** [comment]
> what changes should I make in this issue

it is written above with the marked line number

**Comment by sumanjeet0012** [comment]
I am 

> > what changes should I make in this issue
> 
> it is written above with the marked line number

**Comment by sumanjeet0012** [comment]
@Sanchitbajaj02 This is just a single line code change , I am expecting clear instructions from you about what should I do in this issue.

If this code change is correct accept the PR.
If something needs to be done just give me clear instructions I will do it as soon as possible.
and if this issue is not important then I will start working on some other issues.

 Just provide clear instructions .......

**Comment by Sanchitbajaj02** [comment]
I am approving and merging this PR for now. If anyone faces any issue related to verification, you have to handle it

**Comment by sumanjeet0012** [comment]
> I am approving and merging this PR for now. If anyone faces any issue related to verification, you have to handle it

Yes, of course

**Pull request merged**